### PR TITLE
Microverse 3: Project

### PR DESCRIPTION
**Pull request by Haadiiii**
## Mobile version Portfolio skeleton.

In this project we:

* Add HTML file with texts and images.
* Add CSS file with styling according to Figma template.
* Add Microverse README file and adjust according to the project.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -104,3 +105,705 @@ li {
...
+#click :hover {
+  transition: 0.3s;
+  cursor: pointer;
+}
+
+#one {
+  background-image: url(./images/Placeholder.png);
+  margin-top: 80px;
+  margin-left: 24px;
+}
```
Comment (cwaku) [inline style.css:240]: - [ ] It is a requirement that the content of each project is aligned with the flexbox. Kindly fix that 👍🏾

File: style.css [hunk 2]
```diff
@@ -104,3 +105,705 @@ li {
...
+  width: 327px;
+  height: 48px;
+  text-align: center;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 17px;
+  color: #fff;
+  background-color: #ff6b00;
+  border-color: #ff6b00;
+}
```
Comment (cwaku) [inline style.css:303]: - [ ] I suggest that you add the button interactiveness. Kindly add the effects as required by the design.
- [ ] Also do the same to the buttons in the "About me" section.
image

**Review by cwaku — CHANGES_REQUESTED** [review]
Hi @Haadiiii 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!
There are a few issues 🐛  that you still need to work 🛠️  on to go to the next project ⏭️  but you are almost there! 🏃🏾 

## Highlights
- [x] Very good overall template design 🥇 
- [x] Linters passed 👍🏾 
- [x] Meaningful PR description 💯 
- [x] Correvt use of Github flow ✔️ 


## Required Changes ♻️
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips 🌟  that can improve your code quality 💯 and help you get a better job 💼.

I will guide 📔 you through fixing 🛠️ these bugs, follow my comments down below and make sure you don't miss any of them, in order to get your project approved ✅
- [x] To make your repo more professional. I strongly suggest that you update and fill in the missing parts of your README.
  - Actually, provide the steps to have the project running locally.
  - Provide a description of the project
  - Remove sections that are not necessary for this project.

image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by isaacpitwa — CHANGES_REQUESTED** [review]
Hi @Haadiiii 🙂 ,

Good job so far! I know you have worked hard on implementing the changes requested by the previous reviewer. However, there is an issue that you still need to work on to go to the next project but you are almost there!

Alt Text


## Highlights 
- All linters are passing  ✅
- Correct use of GitHub flow ✅

## Required changes ♻️ 

- [ ] As mentioned by the previous reviewer,  kindly update your README file to achieve a  more professional repository
- Actually, provide the steps to have the project running locally.
- Provide a description of the project
- Remove sections that are not necessary for this project.
      
Check the comments under the review.

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Check the comments under the review.





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @isaacpitwa.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @Haadiiii       , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

[OPTIONAL] Kindly add a screenshot to your README file showing your webpage user Interface. You can use extentions like fireshot to capturer the entire page.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**